packages/terra-infinite-list/src/terra-dev-site/doc/guides/InfiniteListSection.jsx

```diff
@@ -2,14 +2,18 @@ import React from 'react';
 /* eslint-disable import/no-unresolved, import/extensions */
 import InfiniteList, { Item, SectionHeader } from 'terra-infinite-list/lib/index';
 import Placeholder from 'terra-doc-template/lib/Placeholder';
+import classNames from 'classnames/bind';
 /* eslint-enable import/no-unresolved, import/extensions */
 
 import mockData from './mock-data/mock-section';
 import MyExampleContainer from '../common/MyExampleContainer';
+import styles from './InfiniteListDocGuidesCommon.scss';
```

Comment:
This is probably fine, but we should be able to share the examples sass file instead

Reply:
Deleted "InfiniteListDocGuidesCommon.scss" and reused "InfiniteListDocExamplesCommon.scss".